components/centraldashboard/app/api_workgroup.ts

```diff
@@ -209,6 +258,21 @@ export class WorkgroupApi {
                 }
                 res.json(response);
             })
+        .get(
+            '/env-info',
```

Comment:
When we spoke last week, I think this route still has the odd blending of concerns between core Kubernetes information (ie. the platformInfo stuff) and Profiles stuff. I would say that it might be more beneficial to split `/env-info` into different routes so that the non-profiles related information can be kept independent of the profiles information.  It will result in an extra client-side request on page load, but I think that will end up keeping the implementations more narrowly scoped and easier to test and change if need be.